index.html

```diff
@@ -207,7 +216,8 @@ <h2>Scope</h2>
 
                     <li>
                         Work with the Publishing CG's <a href="https://github.com/w3c/publ-a11y/">accessibility
-                        task force</a> to incorporate incubated features.
+                        task force</a> to incorporate incubated features or requirements from legislative changes 
```

Comment:
Should this bullet be split? We don't really need to work with the CG for the addition. We published the EAA mapping directly from the WG, for example.

Maybe it could be something like:

- Develop guidance to help publishers understand legislated ebook accessibility requirements, as appropriate

(But that might be being too non-specific?)